src/internals/irDump.ts

```diff
@@ -0,0 +1,106 @@
+import { CompilationUnit, CFG } from "./ir";
+
+import JSONbig from "json-bigint";
+
+/**
+ * Class responsible for generating a Graphviz dot representation of a CompilationUnit.
+ */
+export class GraphvizDumper {
```

Comment:
Not sure about the quality and features, but there is a bunch of Graphviz-related packages on NPM, like this https://www.npmjs.com/package/graphviz or even TypeScript packages like https://www.npmjs.com/package/ts-graphviz